index.bs

```diff
@@ -116,7 +116,7 @@ The following code queries the available local fonts, and logs the names and met
 ```js
 showLocalFontsButton.onclick = async function() {
   try {
-    const array = await navigator.fonts.query();
+    const array = await self.queryLocalFonts();
```

Comment:
I don't have a strong opinion here, but would developers be more likely to call this as just 

```suggestion
    const array = await queryLocalFonts();
```

? Will that cause them problems such that it's worth using these examples to advise against it?

Reply:
Since eventually the API seemingly ends up on `WindowOrWorkerGlobalScope`, I'd say using `self` (and not `window` or nothing) is the clearest way to convey to developers that the API is defined at the top-level.